packages/beacon-node/src/network/reqresp/score.ts

```diff
@@ -63,5 +63,6 @@ export function onOutgoingReqRespError(e: RequestError, method: ReqRespMethod):
     }
   }
 
+  // other errors like RequestErrorCode.REQUEST_RATE_LIMITED could come from ourself, not the peer so we should not penalize them
```

Comment:
I'm confused about this.  Wouldnt an outgoing request that we "rate limit" mean that we are not making the request.  How would the error "come from ourself"?

Reply:
ahh... as in we throw `RequestErrorCode.REQUEST_RATE_LIMITED` from the client instead of just not making the request?